src/presentation/controller/addressLookUp/Controller.ts

```diff
@@ -335,7 +335,7 @@ class AddressLookUpController extends AbstractController {
     const pageRouting = super.getRouting(addressLookUpRouting, pageType, request);
 
     let cacheKey = "";
-    if (pageType === AddressLookUpPageType.choosePrincipalPlaceOfBusinessAddress) {
+    if (pageType === AddressLookUpPageType.choosePrincipalPlaceOfBusinessAddress || pageType === AddressLookUpPageType.enterPrincipalPlaceOfBusinessAddress || pageType === AddressLookUpPageType.confirmPrincipalPlaceOfBusinessAddress) {
```

Comment:
this if statement is getting a bit long, it would be better to have a separate private method that checks if it is a principalPlaceOfBusinessAddress.
have something like 
`if (isPrincipalPlaceOfBusinessAddressPage(pageType)) {`